content/docs/1.10.0/nodes-and-volumes/volumes/volume-size.md

```diff
@@ -172,3 +172,75 @@ Here we summarize the important things related to disk space usage we have in th
+
+- **Characteristics:**
+  - The `df` command shows 100% usage for the mounted filesystem.
+  - Applications cannot write new data to the volume mount point.
+
+- Example
+
+    ```bash
+    $ df -h /mnt/longhorn-volume-example-dir
+    Filesystem                    Size  Used Avail Use% Mounted on
+    /dev/longhorn-volume-example   12G   12G    0  100% /mnt/longhorn-volume-example-dir
+    ```
+
+#### Node Disk Full
+
+Node disks do not have enough space to accommodate volume operations because volumes are thin-provisioned and the replica sizes on the disks continue to grow.
+
+- Data writes fail with "no space left on device" errors.
+- Volume operations, such as volume creation/expansion, snapshot creation/deletion, and replica rebuilding, may be limited.
+- Replicas of newly created volumes are prevented from being scheduled to the full disks.
+
+- **Characteristics:**
+  - Applications cannot write new data to the volume mount point even though the volume is not full.
+  - Longhorn operations for volumes on these node disks, such as replica creation, rebuilding, or snapshot operations, may fail.
+  - Volumes with replicas sharing the same node disks are affected.
+
+##### Data Integrity Protection for Disks Out-of-Space Scenarios
+
+When multiple replicas simultaneously encounter "no space left on device" errors, Longhorn implements data integrity protection. If all writable replicas are on disks that are out of space, the system preserves the maximum number of replicas that have written the same amount of data (the highest written bytes count). This ensures data consistency by:
+
+1. **Retaining** the maximum number of replicas that have written the same amount of data.  
+This avoids massive replica rebuilding operations, allows for efficient rebuilding when node disks recover from out-of-space issues and ensures that users can still read data consistently.
+2. **Marking** other replicas as **ERR** to prevent data corruption.  
+This ensures that users can read data consistently from retained replicas, and replicas marked as `ERR` will be rebuilt from retained replicas to maintain data integrity when node disks recover from out-of-space issues.
+3. **Maintaining** volume accessibility even when all underlying disks are full.
+
+For example, if replicas A, B, and C encounter space errors after writing 1MB, 2MB, and 2MB respectively, replicas B and C (having written the most data consistently) will remain active while replica A is marked as **ERR**.
+
+### Check And Recovery
+
+- **Check volume filesystem usage**
+
+    ```bash
+    df -h /mnt/your-volume
+    ```
+
+- **Check Longhorn node disk usage**
+
+    ```bash
+    # Check through Longhorn UI: Node -> Node Disk
+    # Or check the data path mount point
+    df -h /var/lib/longhorn # default data path
+    ```
+
+#### Recovery
+
+  1. Scale down the workload.
+  2. Remove unnecessary files on the disk or extend the disk.
```

Comment:
I believe users want to know how to do this step. Can we link this to a reference doc page? If there isn't a suitable one, it's better to create a new one for it.